docs/remove_rhts.rst

```diff
@@ -0,0 +1,278 @@
+Guide to removing RHTS from Jobs
+================================
+
+For some products, Test Requirements include running restraint by itself.
+This requires the exclusion of the legacy RHTS package or `restraint-rhts`
+package installation.  Below lists areas to draw attention in order
+to eliminate RHTS references.
+
+#. Define only `restraint` and not `restraint-rhts` as your harness in your job's
+   recipe definition.  The recipe line below illustrates this harness setting::
+
+     <recipe whiteboard="" role="RECIPE_MEMBERS" **ks_meta="harness='restraint'"**
+         kernel_options="" kernel_options_post="">
```

Comment:
This is Beaker specific actually. If you will use restraint-client + restraintd. Then it depends on how user installed restraint. dnf install restraint or dnf install restraint-rhts?

Reply:
I've made this more general.